scripts/getReleasesInformation.mjs

```diff
@@ -21,34 +21,16 @@ const getCollection = async () => {
   return data
 }
 
-const filterReleases = (data) => {
-  return data.map((release) => {
-    // if repo.name is not in PROJECTS_TO_TRACK, skip it
-    if (release.prerelease)
-      return null
-    if (release.draft)
-      return null
-    return release
-  })
-}
-
 (async () => {
   const collection = await getCollection()
-  const releases = filterReleases(collection)
-
-  // remove nulls from repos array
-  // const filteredReposData = repos.filter(Boolean)
-  // const repoContrinutors = await Promise.all(filteredReposData.map(async (r) => {
-  //   const response = await fetch(`https://api.github.com/repos/PlayCover/${r.name}/contributors?anon=1`)
-  //   const contributors = await response.json()
-  //   return contributors
-  // }))
+  // Remove any null/invalid elements in the collection
+  const releases = collection.filter(Boolean)
```

Comment:
This will pass everything, since it github wont return null values. it will also not filter drafts